android/app/src/main/res/values/string-arrays.xml

```diff
@@ -24,6 +24,17 @@
     <item>1</item>
   </string-array>
 
+  <string-array name="map_languages">
```

Comment:
All supported map languages are defined in coding/string_utf8_multilang.cpp, they're not duplicated anywhere else, right? It would be great to avoid any duplication.

Reply:
This is code from my early experiments. It's not used in current version. Removed in last commit.